cmake/Program.cs

```diff
@@ -202,14 +205,14 @@ private static void serialize()
                     0,
                     0,
                     localTime3dCrs);
-                w1i1SeismicFrameRep.setMdValues(logMds.cast(), 5, hdfProxy);
+                w1i1SeismicFrameRep.setMdValues(logMds.cast(), 5);
```

Comment:
logMds should be GC kept alive in order to prevent to be collected before setMdValues finishes.

Reply:
Done.